HACKING.md

```diff
@@ -234,63 +234,41 @@ the build frontend is [build](https://pypi.org/project/build/).
-    - in [pyproject.toml for `ops-scenario`](testing/pyproject.toml), the `version` attribute and the required version for `ops`
-    - in [pyproject.toml for `ops-tracing`](tracing/pyproject.toml), the `version` attribute and the required version for `ops`
-11. Add, commit, and push, and open a PR to get the `CHANGES.md` update and version
-   bumps into main (and get it merged).
-12. Wait until the tests pass after the PR is merged. It takes around 10 minutes.
-   If the tests don't pass at the tip of the main branch, do not release.
-13. When you are ready, click "Publish". GitHub will create the additional tag.
-
-    Pushing the tags will trigger automatic builds for the Python packages and
-    publish them to PyPI ([ops](https://pypi.org/project/ops/) and
-    [ops-scenario](https://pypi.org/project/ops-scenario)) (authorisation is handled
-    via a [Trusted Publisher](https://docs.pypi.org/trusted-publishers/) relationship).
-    Note that it sometimes take a bit of time for the new releases to show up.
-
-    See [.github/workflows/publish-ops.yaml](.github/workflows/publish-ops.yaml) and
-    [.github/workflows/publish-ops-scenario.yaml](.github/workflows/publish-ops-scenario.yaml) for details.
-    (Note that the versions in the YAML refer to versions of the GitHub actions, not the versions of the ops  library.)
-
-    You can troubleshoot errors on the [Actions Tab](https://github.com/canonical/operator/actions).
-14. Announce the release on [Discourse](https://discourse.charmhub.io/c/framework/42)
-    and [Matrix](https://matrix.to/#/#charmhub-charmdev:ubuntu.com).
-15. Open a PR to change the version strings to the expected next version, with ".dev0" appended.
-   For example, if 2.90.0 is the next expected `ops` version, use
-   `ops==2.90.0.dev0 ops-tracing==2.90.0.dev0 ops-scenario==7.90.0.dev0`.
-   There will be a total of seven changes:
-    - in [pyroject.toml for `ops`](pyproject.toml), the required versions for `ops-scenario` and `ops-tracing`
-    - in [ops/version.py for `ops`](ops/version.py), the version declared in the `version` variable
-    - in [pyproject.toml for `ops-scenario`](testing/pyproject.toml), the `version` attribute and the required version for `ops`
-    - in [pyproject.toml for `ops-tracing`](tracing/pyproject.toml), the `version` attribute and the required version for `ops`
+1. Draft a release: Run: `tox -e draft-release`.
+
+    > This assumes a draft release on the main branch, and your forked remote name is `origin`, and the `canonical/operator` remote name is `upstream`.
+    > 
+    > If you have different settings, add parameters accordingly. For example, the following command assumes your forked remote name is `mine`, and `canonical/operator` remote name is `origin`:
+    > 
+    > `tox -e draft-release -- --canonical-remote origin --fork-remote mine`
+    > 
+    > If you want to make a release on another branch, for example, on "2.23-maintenance", run:
```

Comment:
Do I still run the command with my local clone in the main branch? Or do I switch/checkout the branch I want to release first? Or will the script take care of it and it doesn't matter?